src/Orleans.Core.Abstractions/GrainFeature.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Orleans
+{
+    public class GrainInterfaceMetadata
```

Comment:
hmm, strange that you don't get warnings for the missing XML comments

Reply:
Ah yes, I need to fix that. And it is strange... I wonder why? Don't we have WarningAsError enabled, too?